terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/validation/ValidationController.java

```diff
@@ -0,0 +1,164 @@
+/*

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...